src/Fable.React/Fable.Helpers.React.fs

```diff
@@ -755,22 +804,21 @@ let inline ofFunction<[<Pojo>]'P> (f: 'P -> ReactElement) (props: 'P) (children:
 /// Example: `ofImport "Map" "leaflet" { x = 10; y = 50 } []`
 let inline ofImport<[<Pojo>]'P> (importMember: string) (importPath: string) (props: 'P) (children: ReactElement list): ReactElement =
```

Comment:
I was thinking to surround `ofImport`, `ofType`, `ofFunction` with  `#if FABLE_COMPILER` to remove them when compiling on server-side. But this would cause conflicts with IDEs on frontend side, because they don't define `FABLE_COMPILER`.  Not sure, what's the best solution. I guess `ofImport` throws an exception quickly (and users won't probably import a JS file in .NET either) but I don't know what happens with `ofType` and `ofFunction`. Do they throw exception or fail silently? 

The other option would be to extract common parts into a `Fable.React.Core` package and distribute two packages: `Fable.React` for JS side and `Fable.ReactServer` or `Fable.React.Net` 🤔

Reply:
We may need a design guideline for shared libraries between JS and .NET. @MangelMaxime is already working on this: https://github.com/haf/Http.fs/issues/142

Reply:
> I guess ofImport throws an exception quickly (and users won't probably import a JS file in .NET either) but I don't know what happens with ofType and ofFunction. Do they throw exception or fail silently?

I was thinking this problem, too. In my imagination, we can create an help function by wrapping runtime exceptions in a function that only executed in the client side, but not tested yet.

```F#
let hybridView (clientView: 'cilentMode -> ReactElement) (clientModel: 'cilentModel) (serverView: 'serverModel -> ReactElement) (serverModel: 'serverModel) =
#if FABLE_COMPILER
    clientView clientModel
#else
    serverView serverModel
#endif
###

Reply:
And import a js module could write like this:


```F#

#if FABLE_COMPILER
let JsComp: React.ComponentClass<JsCompProps> = importDefault "./jsComp"
#else
let JsComp = Unchecked.defaultof<React.ComponentClass<JsCompProps>>
#endif

```